### ISSUE TITLE
Invalid Requests to CrossRef REST API

### ISSUE THREAD

**Issue by kjw** [lagotto/alm-report#149]
Appears that this service is making requests to the CrossRef API that are invalid. For example, query paramters like:

```
facet=t&filter=from-pub-date%3A2011%2Cuntil-pub-date%3A2015&offset=12526275&order=desc&rows=25&sort=counter_total_all
```

The sort parameter doesn't accept counter_total_all as a value, and this is now rejected by the API under new, stricter validation.

**Comment by mfenner** [comment]
Thanks. The sort parameter `counter_total_all` is for the `plos` search API. Will filter invalid values when calling the CrossRef API.

**Comment by jure** [comment]
Martin, the volume of these requests is around 2000 over the past month. I don't have a plausible theory about how such a number of requests was made, given that there is no obvious way to get that sort parameter into the query, when the app is running in CrossRef mode. Ideas?

**Comment by mfenner** [comment]
Not really clear to me, need to have a closer look at the code. To my knowledge http://parascope.io is the only instance of ALM Report that currently talks to the CrossRef API.

**Comment by kjw** [comment]
Yeah - the requests are coming from `104.131.71.212` which appears to resolve to a Parascope page.
